python/ml4ir/applications/ranking/tests/data/config/model_config.yaml

```diff
@@ -0,0 +1,29 @@
+architecture_key: dnn
```

Comment:
@mohazahran Where do you use this?

Reply:
Honestly I don't know. My code doesn't use it. I believe I copied this model_config from somewhere and added the optimizer and lr schedule part to it.